source/reference/files/ood-portal-yml.rst

```diff
@@ -1,7 +1,7 @@
 .. _ood-portal-generator-configuration:
 
-Configuration
-=============
+ood-portal.yml
```

Comment:
ood_portal.yml should be the title, not ood-portal.yml since the file is ood_portal.yml